can_tools/scrapers/usafacts/data.py

```diff
@@ -30,41 +30,44 @@ def fetch(self) -> pd.DataFrame:
         return pd.read_csv(BASEURL + self.filename)
 
     def normalize(self, data: Any) -> pd.DataFrame:
+        # Make lowercase so they can't change capitalization on us
+        data.columns = [c.lower() for c in data.columns]
+
         # Load data from site and move dates from column names to
         # a new variable
-        cols = ["countyFIPS", "County Name", "State", "stateFIPS"]
-        cols = cols + [c for c in data.columns if c[-2:] in ("20", "21")]
+        cols = ["countyfips", "county name", "state", "statefips"]
+        cols = cols + [c for c in data.columns if ("2020" in c) or ("2021" in c)]
```

Comment:
They also changed how they are formatting their date and they're using a proper date format so we can just check for years now.